packages/shared/locales/en.json

```diff
@@ -683,7 +683,7 @@
         "syncAll": "Sync all",
         "export": "Export",
         "exportNewStronghold": "Export a new Stronghold",
-        "enableDeepLinks": "Enable deep links",
+        "enableDeepLinks": "Enable IOTA links",
```

Comment:
@cvarley100 tbh, I think people maybe more familiar with the term "deep links". If we don't want to use "deep links" we should at least describe them as "`iota://` links". "IOTA links" sounds vague imo

Reply:
The intention is that by calling them IOTA links in Firefly the terminology would become engrained.

Reply:
You wouldn't say `https://` link you would say https link. And `deep links` is not familiar terminology for most users. Whatever we do, it involves teaching users a new phrase - so I thought why not coin a new one?

Reply:
> Whatever we do, it involves teaching users a new phrase - so I thought why not coin a new one?

That's the thing, though — the term "deep linking" is common enough to warrant having a [Wikipedia page](https://en.wikipedia.org/wiki/Mobile_deep_linking). I don't think we should coin a new term when one already exists and is widely accepted